File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/facade/user/UserLikeFacade.java
package com.cakk.domain.mysql.facade.user;

import com.cakk.domain.mysql.annotation.DomainFacade;
import com.cakk.domain.mysql.entity.shop.CakeShop;
import com.cakk.domain.mysql.entity.user.User;

@DomainFacade
public class UserLikeFacade {

	public void likeCakeShop(final User user, final CakeShop cakeShop) {
		user.likeCakeShop(cakeShop);
	}
}

File: cakk-core/src/main/kotlin/com/cakk/core/facade/user/UserLikeFacade.kt
package com.cakk.core.facade.user

import com.cakk.core.annotation.DomainFacade
import com.cakk.infrastructure.persistence.entity.shop.CakeShopEntity
import com.cakk.infrastructure.persistence.entity.user.UserEntity

@DomainFacade
class UserLikeFacade {

    fun likeCakeShop(userEntity: UserEntity, cakeShop: CakeShopEntity) {
        userEntity.likeCakeShop(cakeShop)
    }
}
